packages/vite/src/node/plugins/css.ts

```diff
@@ -2529,6 +2554,16 @@ const makeModernCompilerScssWorker = (
       sassOptions.url = pathToFileURL(options.filename)
       sassOptions.sourceMap = options.enableSourcemap
 
+      const skipRebaseUrls = (unquotedUrl: string, rawUrl: string) => {
+        const isQuoted = rawUrl[0] === '"' || rawUrl[0] === "'"
+        // matches `url($foo)`
+        if (!isQuoted && unquotedUrl[0] === '$') {
+          return true
+        }
+        // matches `url(#{foo})` and `url('#{foo}')`
+        return unquotedUrl.startsWith('#{')
+      }
```

Comment:
https://sass-lang.com/documentation/interpolation/#in-sass-script
https://sass-lang.com/documentation/syntax/special-functions/#url